### PR TITLE
Remove blocking commits from action

### PR DESCRIPTION
**Pull request by paulocsanz**

### CODE REVIEW

File: lib/dal/src/component.rs [hunk 1]
```diff
@@ -2089,41 +2089,6 @@ impl Component {
             .clone_attributes_from(ctx, self.id(), true, true)
             .await?;
 
-        // Enqueue creation actions
```
Comment (britmyerss) [inline lib/dal/src/component.rs:2092]: this is what was causing the duplicate creates - we were enqueuing a create action in both the paste AND on creating the component

Comment (paulocsanz) [reply lib/dal/src/component.rs:2092]: Great catch! I totally missed that

File: lib/dal/src/job/definition/action.rs [hunk 2]
```diff
@@ -128,20 +127,24 @@ impl TryFrom<JobInfo> for ActionJob {
...
 ) -> JobConsumerResult<(Option<DeprecatedActionRunResult>, Vec<String>)> {
+    let span = Span::current();
     let component_id = Action::component_id(ctx, id)
         .await?
         .ok_or(ActionError::ComponentNotFoundForAction(id))?;
 
     let prototype_id = Action::prototype_id(ctx, id).await?;
     let prototype = ActionPrototype::get_by_id(ctx, prototype_id).await?;
-
+    span.record("si.action.kind", &tracing::field::debug(&prototype.kind));
```
Comment (paulocsanz) [inline lib/dal/src/job/definition/action.rs:146]: Nice!

**Review by jhelwig — APPROVED** [review]

**Comment by britmyerss** [comment]
bors merge

**Pull request merged**